edge/services/helloSecretWorld/horizon/service.definition.json

```diff
@@ -0,0 +1,24 @@
+{
+    "org": "$HZN_ORG_ID",
+    "label": "$SERVICE_NAME for $ARCH",
+    "description": "Very simple sample Horizon service that uses a secret",
+    "documentation": "",
+    "url": "$SERVICE_NAME",
+    "version": "$SERVICE_VERSION",
+    "arch": "$ARCH",
+    "sharable": "multiple",
+    "requiredServices": [],
+    "userInput": [],
+    "deployment": {
+        "services": {
+            "ibm.hello-secret": {
+                "image": "${DOCKER_IMAGE_BASE}_$ARCH:$SERVICE_VERSION",
+                "secrets": {
+                    "hw_who": {
+                        "description": "Who to greet"
```

Comment:
instead of Who to greet; secret greeting? I want the word secret in the description somewhere